app/javascript/app/pages/climate-policies/milestones/milestones-styles.scss

```diff
@@ -0,0 +1,88 @@
+@import '~styles/ui-extends.scss';
+@import '~styles/layout.scss';
+
+$iconSize: 60px;
+$borderSize: 5px;
+$correction: 1px;
+
+.page {
+  margin-bottom: 50px;
+}
+
+.titleContainer {
+  display: flex;
+  justify-content: center;
+  background-color: $link-water;
+  border-bottom: 1px solid rgba($jakarta, 0.3);
+  color: $downriver;
+}
+
+.title {
+  padding: 30px;
+  text-transform: uppercase;
+  font-size: $font-size-large;
+  font-weight: bold;
+}
+
+.cell {
+  padding: 20px;
+  color: $downriver;
+  line-height: 1.5;
+}
+
+.statusColumn {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  font-weight: bold;
+  border-right: $borderSize solid $elephant-blue;
+  position: relative;
+  width: 200px;
+}
+
+.table {
+  width: 100%;
+  margin-bottom: 100px;
+}
+
+.row {
+  display: flex;
+
+  > * {
+    flex: 0 1 33%;
+  }
+
+  &:first-child {
+    margin-top: -1px;
+  }
+}
+
+.statusIcon {
+  height: $iconSize;
+  width: $iconSize;
+  position: absolute;
+  right: calc((-1 * (#{$iconSize} / 2 + #{$borderSize} / 2)) + #{$correction}); // icon in the middle of the border
```

Comment:
Calculation ends up with a floating number (`31.5px`) that makes it be misplaced some times. Maybe would be better to round it up to `32px` to avoid this shift.

![image](https://user-images.githubusercontent.com/6906348/50224143-69b92180-039d-11e9-8edf-96e3283672b4.png)